src/status_im/ui/screens/discover/popular_hashtags/views.cljs

```diff
@@ -3,21 +3,39 @@
   (:require [status-im.components.react :as react]
             [status-im.ui.screens.discover.styles :as styles]
             [status-im.ui.screens.discover.components.views :as components]
+            [status-im.components.list.views :as list]
+            [re-frame.core :as re-frame]
```

Comment:
I would expect this require on top :)

Reply:
Why, do we have a specific order? I just auto-complete the require from Cursive.